pkg/iacinventory/iacinventory.go

```diff
@@ -0,0 +1,12 @@
+package iacinventory
+
+import "github.com/soluble-ai/go-jnode"
+
+type IacInventorier interface {
```

Comment:
If it's GithubScanner is a IacInventorier, then it should be IacInventoryScanner I suppose.